client/checkout/stripe-link/index.js

```diff
@@ -69,7 +90,19 @@ const enableStripeLinkPaymentMethod = ( options ) => {
 				'postal_code'
 			);
 		}
+
 		if ( options.complete_billing() ) {
+			if (
+				billingAddress.hasOwnProperty( 'name' ) &&
+				'string' === typeof billingAddress.name &&
+				billingAddress.name.length
+			) {
+				const billingNames = billingAddress.name.split( / (.*)/s, 2 );
+
+				billingAddress.address.last_name = billingNames.splice( -1 );
```

Comment:
Wondering why don;t we use `billingNames[0]` and `billingNames[1]` here?

Reply:
Indeed, we can avoid the split method. thx